spring-xsuaa-it/src/main/java/testservice/api/nohttp/SecurityConfiguration.java

```diff
@@ -13,6 +14,7 @@
 import com.sap.cloud.security.xsuaa.mock.MockXsuaaServiceConfiguration;
 
 @Configuration
+@ConfigurationProperties("xsuaa")
```

Comment:
This shouldn't be required, otherwise we need to inform customers about this incompatible change, in case they define custom security configurations in their productive or testing environment